apps/pie-docs/src/_utilities/markdown.js

```diff
@@ -1,8 +1,33 @@
 const MarkdownIt = require('markdown-it');
 const markdownItAttrs = require('markdown-it-attrs');
+const slugify = require('slugify');
+const anchor = require('markdown-it-anchor');
+const pieIconsSvg = require('../_11ty/filters/pieIconsSvg');
+
+const AnchorIcon = pieIconsSvg({
+    name: 'link',
+    attrs: {
+        'aria-hidden': 'true',
+        height: 16,
+        width: 16,
+        fill: 'interactive-brand',
+    },
+});
 
 const md = new MarkdownIt({
     html: true,
-}).use(markdownItAttrs);
+}).use(markdownItAttrs)
+    .use(anchor, {
+        level: [2, 3, 4],
+        permalink: anchor.permalink.linkInsideHeader({
+            symbol: `
+                <span class="is-visuallyHidden">Jump to heading</span>
```

Comment:
is the idea that a screen reader focuses the heading and then the span is read out before they press enter/select the anchor? Just wondering if "Jump to heading" is clear enough etc. Haven't seen this pattern before but it's a cool idea!

Reply:
I think it was mostly an experiment on what I could get to work. The screen reader seems to ignore it anyway so I will remove